ReduxMovieDBTests/State/MainStateTests.swift

```diff
@@ -155,7 +157,10 @@ class ReduxMovieDBTests: XCTestCase {
             return XCTFail()
         }
     }
-    
+
+    func testToggleFavoriteMovie() {}
+    func toggleShowFavorites() {}
```

Comment:
Did you forget these tests? 😅

Reply:
I've added it to avoid a crash in unit tests, I guess. It was the quick fix ) Do you need some implementation there?